components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -238,7 +261,7 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
   }
 
   resetModal(): void {
-    this.expressionForm.reset();
+    this.resetForm();
     this.unparsableMember = false;
     this.duplicateMember = false;
```

Comment:
Replace these two lines with
```
this.resetErrors();
```
Also, please make `resetErrors` private; it is not used outside the file.